src/main/java/com/github/wovnio/wovnjava/Api.java

```diff
@@ -9,28 +9,45 @@
 import java.net.URL;
 import java.net.URLEncoder;
 import java.net.MalformedURLException;
+import java.net.SocketTimeoutException;
 import java.util.zip.GZIPInputStream;
 import java.util.zip.GZIPOutputStream;
 import java.util.LinkedHashMap;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Collections;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import javax.servlet.http.HttpServletResponse;
 
 import javax.xml.bind.DatatypeConverter;
 
 import net.arnx.jsonic.JSON;
 
 class Api {
+    private static final Map<String, String> wovnFastlyHeaders;
+    static {
+        Map<String, String> headers = new HashMap<String, String>();
+        headers.put("X-Cache", "X-Wovn-Cache");
+        headers.put("X-Cache-Hits", "X-Wovn-Cache-Hits");
+        headers.put("X-Wovn-Surrogate-Key", "X-Wovn-Surrogate-Key");
+        wovnFastlyHeaders = Collections.unmodifiableMap(headers);
```

Comment:
This logic is outside of `Api` concern, I would make `HeadersForwarder` class of it.

Reply:
Extracted methods for setting response headers to a separate class.

Reply:
thanks